src/envoy/mixer/grpc_transport.cc

```diff
@@ -92,9 +92,10 @@ void GrpcTransport<RequestType, ResponseType>::onSuccess(
 
 template <class RequestType, class ResponseType>
 void GrpcTransport<RequestType, ResponseType>::onFailure(
-    Grpc::Status::GrpcStatus status) {
-  ENVOY_LOG(debug, "{} failed with code: {}", descriptor().name(), status);
```

Comment:
can we add ENVOY_LOG(debug, )  here

https://github.com/lizan/proxy/blob/f8ec3d61e37f0a0da4683abf87ff1cad4f759001/src/envoy/mixer/grpc_transport.cc#L61